packages/system/src/color.ts

```diff
@@ -55,23 +54,7 @@ export const color = (props: ColorProps): ResponsiveStyle => {
     const cssValue = props[key as ColorKeys];
     if (cssValue != undefined) {
       const property = colorMappings[key as ColorKeys];
-      const userTheme = theme.getUserTheme();
-      let converter: (value: string | number) => string | number;
-      if (userTheme.colors) {
-        converter = (value) => {
-          if (value in (userTheme.colors ?? {})) {
-            return userTheme.colors?.[value] as string;
-          }
-          return value;
-        };
-      } else {
-        converter = (v) => v;
-      }
-      const responsiveStyles = applyResponsiveStyles(
-        property,
-        cssValue,
-        converter
-      );
```

Comment:
I think that [font](https://github.com/kuma-ui/kuma-ui/blob/main/packages/system/src/font.ts) etc. can also be changed.

Reply:
I see that other things like fontSize have also been added!
I'll check it out and make some additional corrections.